best-practices/content/10_IAM.tex

```diff
@@ -1,13 +1,128 @@
+Der allgemeine Konsenus ist dass ab 15 aktiven Bentuzern ein IAM verwendet werden soll.\\
+
+Eine IAM Lösung bringt viele Vorteile, gleichzeitig birgt es auch Risiken.\\
+
+\begin{minipage}{\linewidth}
+    \begin{multicols}{2}
+        \begin{table}[H]
+            \begin{center}
+                \textbf{Vorteile:}
+                \begin{itemize}
+                    \item zentrales Accountmanagement
+                    \item zentrales Accessmanagement
+                    \item schnelle Skalierung Server/Client Umgebung
+                    \item single Source of Truth\footnote{\href{https://de.wikipedia.org/wiki/Single\_Point\_of\_Truth}{https://de.wikipedia.org/wiki/Single\_Point\_of\_Truth}}
+                    \item Policysystem zur Konfiguration von Client
+                    \item meist gebündelt mit nützlichen Features\footnote{DHCP, DNS, NTP, CA etc. (Produktabhängig)}
+                    \item Single Sign-On
+                \end{itemize}
+            \end{center}
+            \caption{Vorteile IAM}
+        \end{table}
+        \begin{table}[H]
+            \begin{center}
+                \textbf{Nachteile:}
+                \begin{itemize}
+                    \item hoher initial Aufwand
+                    \item höchste Priorität eines Angreifers
+                    \item anfällig auf Misskonfigurationen
+                    \item abhängigkeit eines stabilen Systems
+                \end{itemize}
+            \end{center}
+            \caption{Nachteile IAM}
+        \end{table}
+    \end{multicols}
+\end{minipage}
+%TODO: @sevigrimm ergänzungen?
+%TODO: Add glossary for DHCP DNS NTP CA
+
+\vspace{1cm}
+Wie bei vielem ist es wichtig das richtige Produkt für seine Umgebung zu finden. Als nächstes werden zwei, im Markt beliebte Produkte vorgestellt.
+
+\subsection{Active Directory}
+Active Directory ist ein Produkt von Microsoft und kann exklusiv auf einem Microsoft Windows Server installiert werden. Active Directory ist einfach zu installieren, wobei die Konfiguration manchmal Herausforderungen birgt.\\
+
+Active Directory ist mit mehr als 80\% Marktanteil der Marktführer. Die Active Directory Funktion wird mit DNS, DHCP und NTP gebündelt. Das Produkt bietet alle grundlegenden Funktionen wie User-, Geräteverwaltung, DHCP, DNS, NTP und viele weitere einfache, bis zu komplexe Funktionen. Vorallem in Windowsnetzwerken ist die Integration sehr gut und intuitiv.\\
+
+Active Directory bietet in fast allen Fällen, was ein KMU braucht und viel mehr. Deshalb ist die Wahl von Active Directory für KMUs zu 99\% die richtige Wahle. Es gibt nur wenige sehr spezifische Fälle in denen ein anderes Produkt besseren Nutzen bringt. 
+
+
+\subsection{FreeIPA}
+FreeIPA ist eine Open-Source Implementation eines IAMs, entwickelt durch die Community und Upstream von Red Hat Identity Management. Exklusiv verfügbar auf RPM-based Linux Systemen.\\
```

Comment:
```suggestion
FreeIPA ist eine Open-Source Implementation eines IAMs, entwickelt durch die Community und Upstream von Red Hat Identity Management. Exklusiv verfügbar auf RPM-basierten Linux Systemen.\\
```

Reply:
RPM erklären, evtl beispiele?

Reply:
@sevigrimm  kind of trivial? glossary maybe?